log4j-1.2-api/src/main/java/org/apache/log4j/Priority.java

```diff
@@ -148,6 +149,14 @@ final int getSyslogEquivalent() {
         return syslogEquivalent;
     }
 
+    /**
+     * Returns the Log4j 2.x level associated with this priority
```

Comment:
For getter methods, I write Javadocs that say "Gets ...". IOW, a getter "Gets...", a setter "Sets..." and so on.